### PR TITLE
feat: add configmap for deploy

### PR DESCRIPTION
**Pull request by gxthrj**

### CODE REVIEW

File: samples/deploy/configmap/apisix-ingress-cm.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                            # re-synchronizes with Kubernetes, default is 60s,
+                            # and the minimal resync interval is 30s.
+
+   # APISIX related configurations.
+   apisix:
+     base_url: "http://127.0.0.1:9080/apisix/admin" # the APISIX admin api / manager api
+                                              # base url, it's required.
+kind: ConfigMap
+metadata:
+  name: apisix-ingress-cm
```
Comment (tokers) [inline samples/deploy/configmap/apisix-ingress-cm.yaml:53]: Always add line break af the end of file.

**Comment by tokers** [comment]
@gxthrj The license checker failed.

**Comment by membphis** [comment]
do we need to fix this PR in milestone `0.1.0`?

if yes, please set the right milestone

File: samples/deploy/configmap/apisix-ingress-cm.yaml (hunk 1 above)
Comment (gxthrj) [reply samples/deploy/configmap/apisix-ingress-cm.yaml]: Done

**Comment by tokers** [comment]
LGTM

**Review by tokers — APPROVED** [review]

**Pull request merged**